sympy/utilities/lambdify.py

```diff
@@ -450,6 +455,48 @@ def wrapper(*argsx, **kwargsx):
                     "Expression:\n\n{expr}").format(sig=sig, expr=expr_str)
     return func
 
+def should_dummify(expr, printer=None):
```

Comment:
Why does this function, which returns a boolean, takes a printer as an argument?